File: 23/23.cpp
#include<iostream>
using namespace std;
int main(){
    int start, end;
    cout<<"Enter the range (start, end):"<<endl;
    cin>>start>>end;
    cout<<"Even Numbers:"<<endl;
    for(int i = start;i<=end;i++){
        if(i%2 == 0){
            cout<<i<<" ";
        }
    }
    cout<<"\nOdd Numbers:\n";
    for(int i = start;i<=end;i++){
        if(i%2 == 1){
            cout<<i<<" ";
        }
    }
}

File: 23/23.py
start, end=map(int,input("Enter the range (start and end):").split())
print("Even Numbers:")
for i in range(start, end+1):
    if i%2 == 0:
        print(i, end=" ")
print("\nOdd Numbers:")
for j in range(start, end+1):
    if j%2 == 1:
        print(j, end=" ")
